src/client/components/definition/definition.css

```diff
@@ -0,0 +1,6 @@
+.definition {
```

Comment:
Try:
```
.definition {
  width: 60%;
  margin: 0 0 -1px 0;
  padding: 6px 20px;
  border: 1px grey solid;
}
```

and along with this, add formatting to glossary.css (don't forget to add className & import to the component):
```
.glossary {
  display: flex;
  flex-direction: column;
  align-items: center;
  padding-bottom: 32px;
}
```